src/pages/WordDetail/WordDetail.tsx

```diff
@@ -1,9 +1,11 @@
 import React from 'react';
-
-const WordDetail: React.FC = () => {
+import { RouteComponentProps } from 'react-router';
+const WordDetail: React.FC<RouteComponentProps<{ id: string }>> = (props) => {
+  const { id } = props.match.params;
```

Comment:
그냥 props로는 비구조화 할당이 안되나요?

Reply:
React.FC = () 로 하면 Cannot find name 'props'.라고 뜹니당

Reply:
저의 질문은 `const { id } = props;` 이것이 안되는가에 대한 것이었습니당!

Reply:
id를 리스트 페이지에서 받아오려면 match를 써야 하는 것이 아닌가욤??

Reply:
아.. 파라미터 변수 이름이 props여서 저게 일반 데이터 오브젝트(?) 인줄 알았어요 ㅋㅋㅋㅋ
Props에 Router에서 넘겨주는 웅앵웅 타입의 (이건 저도 잘모름) props가 넘어오고 거기서 비구조화 할당을 해주어야 겠네욤..


```suggestion
const WordDetail: React.FC<RouteComponentProps<{ match: 웅앵웅 }>> = (props) => {
  const { match } = props;
  const { id } = match.params;
```
이런식으로 되어야하지 않나 생각합니다

Reply:
옹 감사합니당 👍